README.md

```diff
@@ -65,3 +65,16 @@ looksSame.createDiff({
 }, function(error) {
 });
 ```
+
+## Building diff image as a Buffer
+
+If you need to do something further with the diff, you must not write the diff on disk and then read it, you should only set the `save` option to `false` and a second parameter will be passed to the callback, a raw `buffer`. By default, the diff will be saved on the disk.
```

Comment:
Proposal: If you don't want the diff image to be written on disk, then you can pass `{save: false}` to the `createDiff` method. The callback will then receive a `Buffer` containing the diff as the 2nd argument. Default is to save the image on disk.